*** ./smali/android/telephony/SmsManager.smali	2015-09-23 10:11:55.165950486 +0800
--- /home/baohailong/patchrom/fixwifi/temp/new_smali/telephony-common.jar.out/./smali/android/telephony/SmsManager.smali	2015-09-23 10:13:21.483185446 +0800
***************
*** 189,194 ****
--- 189,204 ----
      return-object v0
  .end method
  
+ .method public static getDefault(I)Landroid/telephony/SmsManager;
+     .locals 1
+     .param p0, "slotId"    # I
+ 
+     .prologue
+     sget-object v0, Landroid/telephony/SmsManager;->sInstance:Landroid/telephony/SmsManager;
+ 
+     return-object v0
+ .end method
+ 
  
  # virtual methods
  .method public copyMessageToIcc([B[BI)Z
***************
*** 978,980 ****
--- 988,1026 ----
  
      goto :goto_0
  .end method
+ 
+ .method public sendMultipartTextMessage(Ljava/lang/String;Ljava/lang/String;Ljava/util/ArrayList;Ljava/util/ArrayList;Ljava/util/ArrayList;IZI)V
+     .locals 0
+     .param p1, "destinationAddress"    # Ljava/lang/String;
+     .param p2, "scAddress"    # Ljava/lang/String;
+     .param p6, "priority"    # I
+     .param p7, "isExpectMore"    # Z
+     .param p8, "validityPeriod"    # I
+     .annotation system Ldalvik/annotation/Signature;
+         value = {
+             "(",
+             "Ljava/lang/String;",
+             "Ljava/lang/String;",
+             "Ljava/util/ArrayList",
+             "<",
+             "Ljava/lang/String;",
+             ">;",
+             "Ljava/util/ArrayList",
+             "<",
+             "Landroid/app/PendingIntent;",
+             ">;",
+             "Ljava/util/ArrayList",
+             "<",
+             "Landroid/app/PendingIntent;",
+             ">;IZI)V"
+         }
+     .end annotation
+ 
+     .prologue
+     .local p3, "parts":Ljava/util/ArrayList;, "Ljava/util/ArrayList<Ljava/lang/String;>;"
+     .local p4, "sentIntents":Ljava/util/ArrayList;, "Ljava/util/ArrayList<Landroid/app/PendingIntent;>;"
+     .local p5, "deliveryIntents":Ljava/util/ArrayList;, "Ljava/util/ArrayList<Landroid/app/PendingIntent;>;"
+     invoke-virtual/range {p0 .. p5}, Landroid/telephony/SmsManager;->sendMultipartTextMessage(Ljava/lang/String;Ljava/lang/String;Ljava/util/ArrayList;Ljava/util/ArrayList;Ljava/util/ArrayList;)V
+ 
+     return-void
+ .end method
