*** ./smali/com/android/internal/telephony/uicc/SIMRecords.smali	2015-09-23 10:11:56.753974650 +0800
--- /home/baohailong/patchrom/fixwifi/temp/new_smali/telephony-common.jar.out/./smali/com/android/internal/telephony/uicc/SIMRecords.smali	2015-09-23 10:13:23.075206805 +0800
***************
*** 978,983 ****
--- 978,991 ----
  
      iput-object v0, p0, Lcom/android/internal/telephony/uicc/SIMRecords;->mAdnCache:Lcom/android/internal/telephony/uicc/AdnRecordCache;
  
+     new-instance v0, Lcom/android/internal/telephony/uicc/MiuiAdnRecordCache;
+ 
+     iget-object v1, p0, Lcom/android/internal/telephony/uicc/SIMRecords;->mFh:Lcom/android/internal/telephony/uicc/IccFileHandler;
+ 
+     invoke-direct {v0, v1}, Lcom/android/internal/telephony/uicc/MiuiAdnRecordCache;-><init>(Lcom/android/internal/telephony/uicc/IccFileHandler;)V
+ 
+     iput-object v0, p0, Lcom/android/internal/telephony/uicc/SIMRecords;->mMiuiAdnCache:Lcom/android/internal/telephony/uicc/MiuiAdnRecordCache;
+ 
      new-instance v0, Lcom/android/internal/telephony/uicc/VoiceMailConstants;
  
      invoke-direct {v0}, Lcom/android/internal/telephony/uicc/VoiceMailConstants;-><init>()V
***************
*** 1523,1528 ****
--- 1531,1540 ----
  
      invoke-virtual {v0}, Lcom/android/internal/telephony/uicc/AdnRecordCache;->reset()V
  
+     iget-object v0, p0, Lcom/android/internal/telephony/uicc/SIMRecords;->mMiuiAdnCache:Lcom/android/internal/telephony/uicc/MiuiAdnRecordCache;
+ 
+     invoke-virtual {v0}, Lcom/android/internal/telephony/uicc/MiuiAdnRecordCache;->reset()V
+ 
      invoke-virtual {p0}, Lcom/android/internal/telephony/uicc/SIMRecords;->fetchSimRecords()V
  
      :goto_0
***************
*** 7233,7238 ****
--- 7245,7254 ----
  
      invoke-static {v1, v0}, Landroid/os/SystemProperties;->set(Ljava/lang/String;Ljava/lang/String;)V
  
+     const-string v1, "gsm.apn.sim.operator.numeric"
+ 
+     invoke-direct {p0, v1, v0}, Lcom/android/internal/telephony/uicc/SIMRecords;->setSystemProperty(Ljava/lang/String;Ljava/lang/String;)V
+ 
      :goto_0
      iget-object v1, p0, Lcom/android/internal/telephony/uicc/SIMRecords;->mImsi:Ljava/lang/String;
  
***************
*** 7453,7458 ****
--- 7469,7478 ----
  
      invoke-virtual {v0}, Lcom/android/internal/telephony/uicc/AdnRecordCache;->reset()V
  
+     iget-object v0, p0, Lcom/android/internal/telephony/uicc/SIMRecords;->mMiuiAdnCache:Lcom/android/internal/telephony/uicc/MiuiAdnRecordCache;
+ 
+     invoke-virtual {v0}, Lcom/android/internal/telephony/uicc/MiuiAdnRecordCache;->reset()V
+ 
      const-string v0, "SIMRecords: onRadioOffOrNotAvailable set \'gsm.sim.operator.numeric\' to operator=null"
  
      invoke-virtual {p0, v0}, Lcom/android/internal/telephony/uicc/SIMRecords;->log(Ljava/lang/String;)V
***************
*** 7461,7466 ****
--- 7481,7490 ----
  
      invoke-static {v0, v1}, Landroid/os/SystemProperties;->set(Ljava/lang/String;Ljava/lang/String;)V
  
+     const-string v0, "gsm.apn.sim.operator.numeric"
+ 
+     invoke-direct {p0, v0, v1}, Lcom/android/internal/telephony/uicc/SIMRecords;->setSystemProperty(Ljava/lang/String;Ljava/lang/String;)V
+ 
      const-string v0, "gsm.sim.operator.alpha"
  
      invoke-static {v0, v1}, Landroid/os/SystemProperties;->set(Ljava/lang/String;Ljava/lang/String;)V
***************
*** 8352,8356 ****
--- 8376,8416 ----
  
      move-result-object v0
  
+     return-object v0
+ .end method
+ 
+ .method private setSystemProperty(Ljava/lang/String;Ljava/lang/String;)V
+     .locals 2
+     .param p1, "key"    # Ljava/lang/String;
+     .param p2, "val"    # Ljava/lang/String;
+ 
+     .prologue
+     invoke-static {}, Lmiui/telephony/MultiSimManager;->getInstance()Lmiui/telephony/MultiSimManager;
+ 
+     move-result-object v0
+ 
+     const/4 v1, 0x0
+ 
+     invoke-virtual {v0, p1, v1, p2}, Lmiui/telephony/MultiSimManager;->setTelephonyProperty(Ljava/lang/String;ILjava/lang/String;)V
+ 
+     return-void
+ .end method
+ 
+ .method private getSystemProperty(Ljava/lang/String;Ljava/lang/String;)Ljava/lang/String;
+     .locals 2
+     .param p1, "key"    # Ljava/lang/String;
+     .param p2, "def"    # Ljava/lang/String;
+ 
+     .prologue
+     invoke-static {}, Lmiui/telephony/MultiSimManager;->getInstance()Lmiui/telephony/MultiSimManager;
+ 
+     move-result-object v0
+ 
+     const/4 v1, 0x0
+ 
+     invoke-virtual {v0, p1, v1, p2}, Lmiui/telephony/MultiSimManager;->getTelephonyProperty(Ljava/lang/String;ILjava/lang/String;)Ljava/lang/String;
+ 
+     move-result-object v0
+ 
      return-object v0
  .end method
