*** ./smali/android/media/MediaMetadataEditor.smali	2015-09-23 10:10:54.868988951 +0800
--- /home/baohailong/patchrom/fixwifi/temp/new_smali/framework.jar.out/./smali/android/media/MediaMetadataEditor.smali	2015-09-23 10:12:24.534388287 +0800
***************
*** 140,145 ****
--- 140,151 ----
  
      sget-object v0, Landroid/media/MediaMetadataEditor;->METADATA_KEYS_TYPE:Landroid/util/SparseIntArray;
  
+     const/16 v1, 0x3e8
+ 
+     invoke-virtual {v0, v1, v2}, Landroid/util/SparseIntArray;->put(II)V
+ 
+     sget-object v0, Landroid/media/MediaMetadataEditor;->METADATA_KEYS_TYPE:Landroid/util/SparseIntArray;
+ 
      const/16 v1, 0x64
  
      invoke-virtual {v0, v1, v5}, Landroid/util/SparseIntArray;->put(II)V
