*** ./smali/android/database/sqlite/SQLiteCursor.smali	2015-09-23 10:11:39.293705791 +0800
--- /home/baohailong/patchrom/fixwifi/temp/new_smali/framework.jar.out/./smali/android/database/sqlite/SQLiteCursor.smali	2015-09-23 10:13:04.314952004 +0800
***************
*** 594,599 ****
--- 594,603 ----
      :cond_0
      invoke-direct {p0, p2}, Landroid/database/sqlite/SQLiteCursor;->fillWindow(I)V
  
+     iget-object v0, p0, Landroid/database/sqlite/SQLiteCursor;->mWindow:Landroid/database/CursorWindow;
+ 
+     invoke-static {p0, v0, p1, p2}, Landroid/database/sqlite/SQLiteCursorInjector;->middle_onMove_calib_count(Landroid/database/sqlite/SQLiteCursor;Landroid/database/CursorWindow;II)V
+ 
      :cond_1
      const/4 v0, 0x1
  
