*** ./smali/com/android/internal/app/IAppOpsService.smali	2015-09-23 10:12:05.842111845 +0800
--- /home/baohailong/patchrom/fixwifi/temp/new_smali/framework2.jar.out/./smali/com/android/internal/app/IAppOpsService.smali	2015-09-23 10:13:32.943338116 +0800
***************
*** 23,28 ****
--- 23,36 ----
      .end annotation
  .end method
  
+ .method public abstract checkOperationInternal(IILjava/lang/String;)I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract checkPackage(ILjava/lang/String;)I
      .annotation system Ldalvik/annotation/Throws;
          value = {
***************
*** 89,94 ****
--- 97,110 ----
      .annotation system Ldalvik/annotation/Throws;
          value = {
              Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
+ .method public abstract registerCallback(Lcom/android/internal/app/IOpsCallback;)I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
          }
      .end annotation
  .end method
