*** ./smali/android/view/Surface.smali	2015-09-23 10:11:09.737234268 +0800
--- /home/baohailong/patchrom/fixwifi/temp/new_smali/framework.jar.out/./smali/android/view/Surface.smali	2015-09-23 10:12:37.710578633 +0800
***************
*** 344,349 ****
--- 344,377 ----
      .end packed-switch
  .end method
  
+ .method public static screenshot(II)Landroid/graphics/Bitmap;
+     .locals 1
+     .param p0, "width"    # I
+     .param p1, "height"    # I
+ 
+     .prologue
+     invoke-static {p0, p1}, Landroid/view/SurfaceControl;->screenshot(II)Landroid/graphics/Bitmap;
+ 
+     move-result-object v0
+ 
+     return-object v0
+ .end method
+ 
+ .method public static screenshot(IIII)Landroid/graphics/Bitmap;
+     .locals 1
+     .param p0, "width"    # I
+     .param p1, "height"    # I
+     .param p2, "minLayer"    # I
+     .param p3, "maxLayer"    # I
+ 
+     .prologue
+     invoke-static {p0, p1, p2, p3}, Landroid/view/SurfaceControl;->screenshot(IIII)Landroid/graphics/Bitmap;
+ 
+     move-result-object v0
+ 
+     return-object v0
+ .end method
+ 
  .method private setNativeObjectLocked(I)V
      .locals 2
      .param p1, "ptr"    # I
