*** ./smali/com/android/internal/telephony/Connection.smali	2015-09-23 10:11:58.157995966 +0800
--- /home/baohailong/patchrom/fixwifi/temp/new_smali/telephony-common.jar.out/./smali/com/android/internal/telephony/Connection.smali	2015-09-23 10:13:24.483225653 +0800
***************
*** 21,26 ****
--- 21,28 ----
  
  .field protected mCnapNamePresentation:I
  
+ .field private mFirewallCode:I
+ 
  .field mUserData:Ljava/lang/Object;
  
  
***************
*** 46,51 ****
--- 48,57 ----
  
      iput v0, p0, Lcom/android/internal/telephony/Connection;->mCnapNamePresentation:I
  
+     const/4 v0, 0x0
+ 
+     iput v0, p0, Lcom/android/internal/telephony/Connection;->mFirewallCode:I
+ 
      return-void
  .end method
  
***************
*** 104,109 ****
--- 110,124 ----
  .method public abstract getDurationMillis()J
  .end method
  
+ .method public getFirewallCode()I
+     .locals 1
+ 
+     .prologue
+     iget v0, p0, Lcom/android/internal/telephony/Connection;->mFirewallCode:I
+ 
+     return v0
+ .end method
+ 
  .method public abstract getHoldDurationMillis()J
  .end method
  
***************
*** 125,130 ****
--- 140,178 ----
  .method public abstract getRemainingPostDialString()Ljava/lang/String;
  .end method
  
+ .method public getRingDurationMillis()J
+     .locals 6
+ 
+     .prologue
+     invoke-virtual {p0}, Lcom/android/internal/telephony/Connection;->getCreateTime()J
+ 
+     move-result-wide v2
+ 
+     .local v2, "createTime":J
+     invoke-virtual {p0}, Lcom/android/internal/telephony/Connection;->getConnectTime()J
+ 
+     move-result-wide v0
+ 
+     .local v0, "connectionTime":J
+     cmp-long v4, v0, v2
+ 
+     if-lez v4, :cond_0
+ 
+     sub-long v4, v0, v2
+ 
+     :goto_0
+     return-wide v4
+ 
+     :cond_0
+     invoke-static {}, Ljava/lang/System;->currentTimeMillis()J
+ 
+     move-result-wide v4
+ 
+     sub-long/2addr v4, v2
+ 
+     goto :goto_0
+ .end method
+ 
  .method public getState()Lcom/android/internal/telephony/Call$State;
      .locals 2
  
***************
*** 184,189 ****
--- 232,246 ----
      return v0
  .end method
  
+ .method public isForwarded()Z
+     .locals 1
+ 
+     .prologue
+     const/4 v0, 0x0
+ 
+     return v0
+ .end method
+ 
  .method public abstract isIncoming()Z
  .end method
  
***************
*** 216,221 ****
--- 273,288 ----
      .end annotation
  .end method
  
+ .method public setFirewallCode(I)V
+     .locals 0
+     .param p1, "firewallCode"    # I
+ 
+     .prologue
+     iput p1, p0, Lcom/android/internal/telephony/Connection;->mFirewallCode:I
+ 
+     return-void
+ .end method
+ 
  .method public setUserData(Ljava/lang/Object;)V
      .locals 0
      .param p1, "userdata"    # Ljava/lang/Object;
