*** ./smali/android/webkit/WebView.smali	2015-09-23 10:10:36.568679235 +0800
--- /home/baohailong/patchrom/fixwifi/temp/new_smali/framework.jar.out/./smali/android/webkit/WebView.smali	2015-09-23 10:12:08.186146931 +0800
***************
*** 167,172 ****
--- 167,176 ----
  
      invoke-static {}, Landroid/webkit/CookieSyncManager;->setGetInstanceIsAllowed()V
  
+     invoke-static {p0, p1}, Landroid/webkit/WebViewInjector;->initEgretLoader(Landroid/webkit/WebView;Landroid/content/Context;)V
+ 
+     invoke-static {p1}, Lmiui/content/res/ThemeFontChangeHelper;->markWebViewCreated(Landroid/content/Context;)V
+ 
      return-void
  
      :cond_1
