*** ./smali/android/net/wifi/WifiConfiguration$Protocol.smali	2015-09-23 10:11:00.321079553 +0800
--- /home/baohailong/patchrom/fixwifi/temp/new_smali/framework.jar.out/./smali/android/net/wifi/WifiConfiguration$Protocol.smali	2015-09-23 10:12:29.474460083 +0800
***************
*** 29,35 ****
      .locals 3
  
      .prologue
!     const/4 v0, 0x2
  
      new-array v0, v0, [Ljava/lang/String;
  
--- 29,35 ----
      .locals 3
  
      .prologue
!     const/4 v0, 0x3
  
      new-array v0, v0, [Ljava/lang/String;
  
***************
*** 45,50 ****
--- 45,56 ----
  
      aput-object v2, v0, v1
  
+     const/4 v1, 0x2
+ 
+     const-string v2, "WAPI"
+ 
+     aput-object v2, v0, v1
+ 
      sput-object v0, Landroid/net/wifi/WifiConfiguration$Protocol;->strings:[Ljava/lang/String;
  
      return-void
