*** ./smali/android/accounts/ChooseTypeAndAccountActivity.smali	2015-09-23 10:11:26.665506902 +0800
--- /home/baohailong/patchrom/fixwifi/temp/new_smali/framework.jar.out/./smali/android/accounts/ChooseTypeAndAccountActivity.smali	2015-09-23 10:12:52.530788398 +0800
***************
*** 818,828 ****
      :cond_0
      new-instance v0, Landroid/content/Intent;
  
!     const-class v1, Landroid/accounts/ChooseAccountTypeActivity;
! 
!     invoke-direct {v0, p0, v1}, Landroid/content/Intent;-><init>(Landroid/content/Context;Ljava/lang/Class;)V
  
      .local v0, "intent":Landroid/content/Intent;
      const/high16 v1, 0x80000
  
      invoke-virtual {v0, v1}, Landroid/content/Intent;->setFlags(I)Landroid/content/Intent;
--- 818,828 ----
      :cond_0
      new-instance v0, Landroid/content/Intent;
  
!     invoke-direct {v0}, Landroid/content/Intent;-><init>()V
  
      .local v0, "intent":Landroid/content/Intent;
+     invoke-static {v0}, Landroid/accounts/ChooseTypeAndAccountActivityInjector;->toMiuiChooseAccountTypeActivity(Landroid/content/Intent;)V
+ 
      const/high16 v1, 0x80000
  
      invoke-virtual {v0, v1}, Landroid/content/Intent;->setFlags(I)Landroid/content/Intent;
***************
*** 1037,1042 ****
--- 1037,1046 ----
      .local v4, "accountType":Ljava/lang/String;
      if-eqz v4, :cond_5
  
+     const/4 v11, 0x1
+ 
+     iput v11, p0, Landroid/accounts/ChooseTypeAndAccountActivity;->mPendingRequest:I
+ 
      invoke-virtual {p0, v4}, Landroid/accounts/ChooseTypeAndAccountActivity;->runAddAccountForAuthenticator(Ljava/lang/String;)V
  
      goto :goto_1
